index.html

```diff
@@ -5,29 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title><Nafees-Portfolio></Nafees-Portfolio></title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.cdnfonts.com/css/crete-round" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+                
 </head>
 <body>
-     <nav>
-            <ul class="nav-bar">
-                <li class="logo"><a href="#"><img src="NGeelani logo.png" alt="Logo"></a></li>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
-                <li><a href="#"><img src="Rec.png" alt="Menu"></a></li>
-            </ul>
-        </nav>
+     <header class="nav-bar">
+                <a class="logo" href="#">Muhammad Nafees</a>
+                <nav class="menu" href="#"><img src="Union.png" alt="Menu bar"></nav>
+     </header>
         <section class="content">
-    <h1 class="my-name">Hey There I'm Nafees</h1>
-    <h2 class="profession">I’m a Software Developer</h2>
-    <p class="i-can-do">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    <h1 class="my-name">Hey There.<br/> I'm Nafees</h1>
+    <h3 class="profession">I’m a Software Developer</h3>
+    <p class="i-can-do">I can help you build a product , <br /> feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hesitate to contact me.</p>
-        </section>
         <ul class="links">
             <li><a title="GitHub" href="https://github.com/ngeelani48"><img src="github.png" alt="GitHub Profile"></a></li>
             <li><a title="Twitter" href="https://twitter.com/ngeelani48"><img src="twitter.png" alt="Twitter Profile"></a></li>
             <li><a title="Medium" href="https://medium.com/@engnrnafees"><img src="medium.png" alt="Medium Profile"></a></li>
             <li><a title="LinkedIn" href="https://www.linkedin.com/in/m-n-geelani-3446097275/"><img src="Linkedin icon.png" alt="LinkedIn Profile"></a></li>
+            <li><a title="GitHub" href="#"><img src="Vector.png" alt="angelist Profile"></a></li>
         </ul>
-        <img src="Illustration-header.png" alt="Portfolio design">
-</body>
+    </section>
```

Comment:
Overall, you did a great job working on this section! 💯  However, there's still some room for improvement. One of the main goals of this project is that you demonstrate the capability to follow as closely as possible a given design, this includes colors, fonts, and spacing. Please try to stick as much as you can to the FIGMA template:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/212162049-a852031e-b5c0-4084-94ed-260b2378bc6f.png) | ![image](https://user-images.githubusercontent.com/77069011/212161882-6fafe2e9-5260-4a6f-bba3-9527b6df0ff7.png) |

- [ ] 🚧 Please consider removing the margin that surrounds your entire page, you could do this by simply setting a `margin: 0;` property to the body element, however, I suggest that you reset the margin and padding of all elements and set the box-sizing to border-box, giving you a better time styling your page: 🔧 

```css
* {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
}
```

- [ ] 🚧 The toolbar's height is too small, please consider increasing it to match the given design! 🔧 

- [ ] 🚧 The Title's font is not the correct one, please consider fixing it! 🔧

- [ ] 🚧 The background image is not showing because there's a typo in the path, please consider looking into it and fixing it! 🔧 

Kindly apply these changes! 🚀 🤓